lib/IonDid.ts

```diff
@@ -15,8 +16,8 @@ export default class IonDid {
    * @param input.document The initial state to be associate with the ION DID to be created using a `replace` document patch action.
    */
   public static createLongFormDid (input: {
-    recoveryKey: JwkEs256k;
-    updateKey: JwkEs256k;
+    recoveryKey: JwkEs256k | JwkEd25519;
```

Comment:
consider moving this or type to a proper type for `SidetreeOperationalPrivateKeyJwk` or similar

Reply:
Added SidetreeKeyJwk type https://github.com/decentralized-identity/ion-sdk/pull/23/commits/bd6625238dc73c9f6ac5962cd3b9d22a87da588a